CHANGES.txt

```diff
@@ -15,9 +15,45 @@ the appropriate section of the docs.
 Breaking Changes
 ================
 
+- Dropped support for tables that have been created with CrateDB prior to
+  version 2.0. Tables which require upgrading are indicated in the cluster
+  checks, including visually shown in the Admin UI, if running the latest 2.2
+  or 2.3 release. The upgrade of tables needs to happend before updating
+  CrateDB to this version. This can be done by exporting the data with ``COPY
+  TO`` and importing it into a new table with ``COPY FROM``.  Alternatively you
+  can use ``INSERT`` with query.
```

Comment:
Should we explain links to explain the two variants in more detail?

Reply:
We could do that in upgrade notes for the release.